openapi-cli/src/test/java/io/ballerina/openapi/generators/client/RemoteFunctionNameValidationTests.java

```diff
@@ -30,7 +30,8 @@
 
     @Test(description = "When path parameter has given unmatch data type in ballerina",
             expectedExceptions = BallerinaOpenApiException.class,
-            expectedExceptionsMessageRegExp = "OperationId is missing for the resource path: .*")
+            expectedExceptionsMessageRegExp = "OpenAPI definition has errors: \n\n" +
```

Comment:
Better if we can use `System.lineSeparator()` there as well for the new line.

Reply:
Could not use `System.lineSeperator()` when building `expectedExceptionsMessageRegExp` since `System.lineSeperator()` is a runtime constant. Used Use `\\R` which is the regexp for any newline. Please review.